cmd/bootstrap/bootstrap.sh

```diff
@@ -0,0 +1,141 @@
+set -Ceu
+
+IMAGE="quay.io/skupper/bootstrap"
```

Comment:
Is the version of this image relevant to the manifest?

Reply:
Certainly! I am leaning towards using the same approach that is used by the install.sh script to
determine the version to be installed.
I will raise a tracker to handle the bootstrap image version in the bootstrap procedure and will
work on it once we have our first official version out.